app/controllers/merchants_controller.rb

```diff
@@ -0,0 +1,13 @@
+class MerchantsController < ApplicationController
+
+  def index
+    @merchants = MerchantFacade.new
```

Comment:
Rather than calling `all_merchants` in the view, I'd do that here so `@merchants` contains the collection of merchants by the time it goes to the view, similar to how you're handling it for the `show` action on lines 10 and 11 below.